hex/Dockerfile

```diff
@@ -6,8 +6,8 @@ RUN apt-get update \
 # Install Erlang and Elixir
 ENV PATH="$PATH:/usr/local/elixir/bin"
 # https://github.com/elixir-lang/elixir/releases
-ARG ELIXIR_VERSION=v1.14.2
-ARG ELIXIR_CHECKSUM=2e4addb85de85218d32c16b3710e8087f5b18b3b1560742137ad4c41bbbea63a
+ARG ELIXIR_VERSION=v1.14.4
+ARG ELIXIR_CHECKSUM=b5705275d62ce3dae172d25d7ea567fffdabb45458abeb29c0189923b907367c
 ARG ERLANG_MAJOR_VERSION=24
 ARG ERLANG_VERSION=1:${ERLANG_MAJOR_VERSION}.2.1-1
```

Comment:
This version is a behind one major upgrade and various patches.

 Can we also upgrade it to the [latest patch](https://www.erlang.org/patches/otp-24.3.4.11)? (in either this PR or separately)